include/envoy/server/filter_config.h

```diff
@@ -295,6 +295,15 @@ class NamedHttpFilterConfigFactory {
    */
   virtual ProtobufTypes::MessagePtr createEmptyConfigProto() { return nullptr; }
 
+  /**
+   * @return ProtobufTypes::MessagePtr create an empty virtual host, route, or weight cluster-local
+   *         config proto message for v2. The filter config, which arrives in an opaque
+   *         google.protobuf.Struct message, will be converted to JSON and then parsed into this
+   *         empty proto. By default, this method returns the same value as createEmptyConfigProto,
+   *         and can be optionally overridden in implementations.
+   */
+  virtual ProtobufTypes::MessagePtr createEmptyRDSConfigProto() { return createEmptyConfigProto(); }
```

Comment:
s/RDS/Route/ ?
The route config can come from static config as well.

Reply:
+1